recommend/report_html.go

```diff
@@ -136,20 +137,39 @@ type RecordInfo struct {
 }
 
 // Record addition of new HTML table row
-func (r HTMLReport) Record(ms MatchSpec, policyName string) error {
+func (r HTMLReport) Record(ms *MatchSpec, policyName string) error {
 	*r.RecordCnt = *r.RecordCnt + 1
 	policy, err := os.ReadFile(filepath.Clean(policyName))
 	if err != nil {
 		log.WithError(err).Error(fmt.Sprintf("failed to read policy %s", policyName))
 	}
 	policyName = policyName[strings.LastIndex(policyName, "/")+1:]
+	var severity, action string
+	if ms.Spec.File.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.File.Severity)
+	} else if ms.Spec.Process.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.Process.Severity)
+	} else if ms.Spec.Network.Severity > 0 {
+		severity = fmt.Sprintf("%d", ms.Spec.Network.Severity)
+	} else {
+		severity = fmt.Sprintf("%d", ms.Spec.Severity)
+	}
+	if ms.Spec.File.Action != "" {
+		action = string(ms.Spec.File.Action)
+	} else if ms.Spec.Process.Action != "" {
+		action = string(ms.Spec.Process.Action)
+	} else if ms.Spec.Network.Action != "" {
+		action = string(ms.Spec.Network.Action)
+	} else {
+		action = string(ms.Spec.Action)
```

Comment:
We have a problem here.
Consider we have two rules, one file and another process, with Severity 1, 2 resp and Action as audit, block resp.

The severity will be decided to be 1 and action will be concluded to be audit.

I am not sure how to handle this.

We cannot show a single severity/action at the policy level if multiple rules have their individual severity/action.

Reply:
yes, i updated the code because it was showing blank for action and severity as 0 if there is no policy-level action/severity. 
should we show multiple severity and action in the table?